app/main.py

```diff
@@ -1,3 +1,69 @@
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int) -> int:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        elif isinstance(other, (int, float)):
+            return Distance(self.km + other)
+        else:
+            return NotImplemented
+
+    def __iadd__(self, other: int) -> int:
+        if isinstance(other, Distance):
+            self.km += other.km
+        elif isinstance(other, (int, float)):
+            self.km += other
+        return self
+
+    def __mul__(self, multiplier: int) -> int:
+        if isinstance(multiplier, (int, float)):
+            return Distance(self.km * multiplier)
+        else:
+            return NotImplemented
+
+    def __truediv__(self, divisor: int) -> int:
+        if isinstance(divisor, (int, float)):
+            if divisor == 0:
+                raise ValueError("Ділення на нуль неможливе.")
+            return Distance(round(self.km / divisor, 2))
+        else:
+            return NotImplemented
+
+    def __lt__(self, other: int) -> int:
```

Comment:
The return type annotation for the `__lt__` method should be `bool`, not `int`, since it returns a boolean value.